test/lib/code_corps/messages/conversation_parts_test.exs

```diff
@@ -8,14 +8,42 @@ defmodule CodeCorps.Messages.ConversationPartsTest do
   }
 
   @valid_attrs %{
-    body: "Test body."
+    body: "Test body.",
+    part_type: "comment"
```

Comment:
You can remove this line and the tests should still pass, with the changes mentioned above.

Reply:
If I set to a default, do I need to do something in the changeset chain to include it in the changes?

Reply:
No it’ll just get set if it’s not set. Remember Ecto is still just an adapter to SQL.